library/globals.inc.php

```diff
@@ -2706,10 +2699,10 @@ function gblTimeZones()
         ),
 
         'fhir_base_url' => array(
-            xl('FHIR Server Base Address'),
+            xl('FHIR Server Base URI'),
             'text',
             'https://hapi.fhir.org/baseDstu3/',
-            xl('Base URL for FHIR Server. Url should end with /')
+            xl('Base URI for FHIR Server. Url should end with /')
```

Comment:
Just making sure you meant this one too :)

Reply:
same